.gitpod.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM gitpod/workspace-full
+
+USER root
+
+RUN apt-get update \
+    && apt-get install -y ruby-full \
+    && apt-get clean && rm -rf /var/cache/apt/* /var/lib/apt/lists/* /tmp/*
+
+USER gitpod
+
+RUN gem install jekyll bundler
```

Comment:
... otherwise you would need to do the `gem install` as `root` user. If that's what you intend to do, please move the line `USER gitpod` below this one.

Reply:
ref. https://github.com/gitpod-io/gitpod/issues/1672